helm/oauth2-proxy/templates/deployment.yaml

```diff
@@ -47,7 +47,9 @@ spec:
         image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
         imagePullPolicy: {{ .Values.image.pullPolicy }}
         args:
+        {{- if not .Values.config.alphaConfigFile }}
           - --http-address=0.0.0.0:4180
+        {{- end }}
```

Comment:
I had a look in the main repo, but I don't understand how this arg is related to the use of `cfg.alpha`. Could you enlighten me?

Reply:
When alpha configuration is in use the flag is not recognised:

[2021/06/07 22:35:41] [main.go:75] WARNING: You are using alpha configuration. The structure in this configuration file may change without notice. You MUST remove conflicting options from your existing configuration.
unknown flag: --http-address

Reply:
Any chance we could merge this?

Reply:
How can we progress this?